add a README

tmieslinger opened the pull request:
the Readme includes
* a description of the repository content
* the usage of the generated docker image

tmieslinger committed:
add a README

hgloeckner commented:
Thanks it's great :) I stumbled a bit over  'how to install docker' because I did it wrong the first try... Maybe we could add a command there on how to easily install it? 
Maybe we can also expect people to know/research that on their own. 
In the end I used 
```
brew install --cask docker
```
and opened docker desktop manually once. 
I also ran 
```
 docker pull ghcr.io/atmdrops/aspenqc
```
before I was able to use the `docker run --help` comment without an error. I don't know if all of that is necessary, but if it is, we might want to add it? What do you think? 
I still get a `WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested`, but that's something I will figure out tomorrow :)

tmieslinger committed:
add info to docker installation and beautify code statement

tmieslinger commented:
Thanks for testing the docker image!

I added a note on the docker installation. The simplest way is via [Docker Desktop](https://docs.docker.com/desktop/) which is what you get from the main docker website (https://www.docker.com/) and also with `brew install --cask docker` for MacOS. I updated the link in the README to point to the main docker website and I also added a note on brew installation

In principle, it would be sufficient to only have [Docker Engine](https://docs.docker.com/engine/), but you'd still have to start `dockerd` (docker daemon) before using the ASPEN docker image.

The `docker pull` should not be necessary. `docker run` will pull the image, too.

The warning message is ok. The docker image uses a linux/amd64 platform and your Mac warns you that it's not their :)
